mvp-spec/audit-log-design.md

```diff
@@ -33,7 +33,7 @@ the Audit UI.
 The Contracting Party responsible for the Audit Log must build a valid Audit Log. 
 For this purpose, it must:
 
-1. Extract the **Transmission Results** from the parents and the children of its Transmission
+1. Extract the list of **Transmission Results** from the parents and the children of its Transmission. The chronological order must be keeped for this final list.
```

Comment:
nit: keeped → kept

Reply:
I will update on the fly in another PR. Thanks.